crates/controller/src/mouse.rs

```diff
@@ -5,6 +5,7 @@ use bevy::{
 };
 use de_core::{screengeom::ScreenRect, stages::GameStage, state::GameState};
 use iyes_loopless::prelude::*;
+use de_hud::HUDInteraction;
```

Comment:
I would change this to `HudInteraction` so only initial letter of the abbreviation is capitalized.